list.txt

```diff
@@ -1,45 +1,65 @@
 4ci-usa.com
-DISYS.COM
+aalgorithm.com
 aasolutionsinc.com
 absiusa.com
 acetechnologies.com
 aclatinc.net
 acmesoftinc.com
 acsicorp.com
 advancedtechno.com
+adventglobal.com
 aerotek.com
+agamasolutions.com
 agilepeoplestaffing.com
 agreeya.com
 aitrg.com
 alissoftware.com
 altaits.com
+alz.org
+amazon.com
```

Comment:
Ah I should probably remove this one, though to be fair Amazon recruiters are insane.

Reply:
You understand that this will filter all emails coming from @amazon.com, right?

Reply:
You understand that this will filter all emails coming from @amazon.com, right?

Reply:
Yeah, I just removed that one.